gramps/gen/plug/utils.py

```diff
@@ -182,6 +185,19 @@ def available_updates():
         context = create_default_context()
         context.check_hostname = False
         context.verify_mode = CERT_NONE
+    timeout = 10 # seconds
+    fp = None
+    try:
+        fp = urlopen(URL, timeout=timeout, context=context)
+    except TypeError:
+        fp = urlopen(URL, timeout=timeout)
+    except Exception as err: # some error
+        LOG.warning("Failed to open URL {url}: {err}".
+            format(url=URL, err=err))
```

Comment:
Don't catch this exception here. Put the urlopen_maybe_no_check_cert() in a try block. That prevents logging a warning just because it's the first locale try, failure of which isn't an error, saves the test of fp there, and allows us to keep the different warning messages for the addons list and the actual addon.